engine/src/flutter/impeller/renderer/backend/gles/proc_table_gles.cc

```diff
@@ -141,6 +141,10 @@ ProcTableGLES::ProcTableGLES(  // NOLINT(google-readability-function-size)
     DiscardFramebufferEXT.Reset();
   }
 
+  if (!description_->HasExtension("GL_ANGLE_framebuffer_blit")) {
+    BlitFramebufferANGLE.Reset();
+  }
```

Comment:
I considered only loading this proc if we detect ANGLE, however, the `GL_VERSION` for the affected devices is `2.0.0`, which causes `DescriptionGLES::IsANGLE` to return `false`.